pyproject.toml

```diff
@@ -56,6 +56,8 @@ tf = [
     # cf. https://github.com/mindee/doctr/pull/1461
     "tensorflow>=2.11.0,<2.16.0",
     "tf2onnx>=1.16.0,<2.0.0",  # cf. https://github.com/onnx/tensorflow-onnx/releases/tag/v1.16.0
+    # TODO: This is a temporary fix until we can upgrade to a newer version of tensorflow
```

Comment:
Temp pin numpy until we can upgrade tensorflow to fix the issue with `poetry` and the failing API test (Will have effect after merging)